utils/show.cc

```diff
@@ -468,11 +599,28 @@ int tr_main(int argc, char* argv[])
         }
         else
         {
+            bool print_ok_files = false;
+            if (opts.print_files && opts.verify)
+            {
+                /* If opts.verify, verify() will print the file list including the status.
+                 * So disable the file list in showInfo(). */
+                print_ok_files = true;
+                opts.print_files = false;
+            }
             showInfo(opts, metainfo);
+
+            if (opts.verify)
+            {
+                bool ok = verify(opts, print_ok_files, metainfo);
+                if (!ok)
+                {
+                    exit_status = EXIT_FAILURE;
+                }
+            }
```

Comment:
To avoid deeply-nested code, looks like this block can be moved up to the same level as `if (opts.print_header)` and `if (opts.scrape)`.

Reply:
I've moved it but I am wondering why `showInfo()` is not called when `opts.scrape` is enabled.

Reply:
I don't see why not. If you're interested, you're welcome to submit another PR for it :)